frontend/app/components/toggle.js

```diff
@@ -1,14 +1,7 @@
-import { assert } from "@ember/debug";
 import { action } from "@ember/object";
 import Component from "@glimmer/component";
 
-export default class SyToggle extends Component {
-  constructor(...args) {
-    super(...args);
-
-    assert("You must pass a onToggle callback.", this.args.onToggle);
-  }
-
```

Comment:
because we don't have checks like this in any other components (datepicker, timepicker etc. to name a few)

Reply:
Question is: Which approach is to favor? 

If you understand your components as some sort of API you serve to the users (other devs making use of the component, although in the same app), it would be nice to have some sort of API validation. On the other hand one could argue that this is to bloated. Decide for your self. :man_shrugging: 

PS: If this would have been a UI library I would have insisted on that assertion.

Reply:
Answer: typescript